src/pages/userProfile.js

```diff
@@ -40,25 +41,65 @@ class Profile extends React.Component {
             console.log("From userProfile.js - error in getting the user's information", err);
         }
 
-        // Attempt to add one applied job for testing
+        // What to pass in when creating applied job objects. The id parameter must be different each time
+        const appliedJobInput = {
+            id: "54160v2d-7469-411e-9b53-012ed15c97d0",
+            dateApplied: "right this MOMENT",
+            status: "got it YET AGAIN",
+            appliedJobEmployeeId: "59560b0d-6069-431e-9b33-079ed12c08f0",   // my user id
+            appliedJobJobId: "54563b0d-7069-411e-9n33-049ed15c02f0"
+        }
+
+        const postedJobInput = {
+            id: "54563b0d-7069-411e-9n33-049ed15c02f0",
+            jobTitle: "Front End Developer",
+            jobType: "Management"
+        }
+        /*
+        // Attempt to add one posted job for testing
         try {
-            const newAppliedJob = await API.graphql(graphqlOperation(mutations.createAppliedJob, {input: this.state.userID}));
-            console.log("From userProfile.js - The test job was added");
+            const newPostedJob = await API.graphql(graphqlOperation(mutations.createPostedJob, {input: postedJobInput}));
+            console.log("From userProfile.js - The test posted job was added");
         } catch(err) {
-            console.log("From userProfile.js - Error: The test job was not added because it already exists");
+            console.log("From userProfile.js - Error: ", err);
         }
 
+        // Attempt to add one applied job for testing
+        try {
+            const newAppliedJob = await API.graphql(graphqlOperation(mutations.createAppliedJob, {input: appliedJobInput}));
+            console.log("From userProfile.js - The test applied job was added");
+        } catch(err) {
+            console.log("From userProfile.js - Error: ", err);
+        }*/
+
         // Fetch all relevant jobs and save to state to render to page
         try {
             // We can fetch an applied job by id now. But now we have to filter it by the employee id which returns results specific to the user
-            let fetchAllAppliedJobs = await API.graphql(graphqlOperation(queries.getAppliedJob, { id: this.state.userID }));
-            if (fetchAllAppliedJobs.data == null) {
+            let fetchAllAppliedJobs = await API.graphql(graphqlOperation(queries.listAppliedJobs));
```

Comment:
right now, you are getting all the job that was applied by all the users. This will be a huge data if there are users and actual jobs.